### PR TITLE
Swift first exception messages

### PR DESCRIPTION
**Pull request by ncooke3**
Changing exception messages to be more Swift oriented.

Follow-up PR to conversation from https://github.com/firebase/firebase-ios-sdk/pull/8266#discussion_r656615037

### CODE REVIEW

**Comment by ncooke3** [comment]
Hmm would this type of public-facing change require CHANGELOG updates?

**Comment by ncooke3** [comment]
`auth / integration-tests` are failing. Issue being tracked at https://github.com/google/GTMAppAuth/issues/128

**Review by paulb777 — APPROVED** [review]
Thanks @ncooke3 !

File: FirebaseAppCheck/Sources/Core/FIRAppCheck.m [hunk 1]
```diff
@@ -172,12 +172,13 @@ - (instancetype)initWithAppName:(NSString *)appName
...
-                format:@"The default FirebaseApp instance must be configured before the default"
-                       @"AppCheck instance can be initialized. One way to ensure that is to "
-                       @"call `[FIRApp configure];` (`FirebaseApp.configure()` in Swift) in the App"
-                       @" Delegate's `application:didFinishLaunchingWithOptions:` "
-                       @"(`application(_:didFinishLaunchingWithOptions:)` in Swift)."];
+    [NSException
+         raise:FIRAppCheckErrorDomain
+        format:@"The default FirebaseApp instance must be configured before the default"
+               @"AppCheck instance can be initialized. One way to ensure that is to "
+               @"call `FirebaseApp.configure()` (`[FIRApp configure];` in Objective-C) in the App"
```
[the remark this answers is not in the document]
Comment (ncooke3) [reply FirebaseAppCheck/Sources/Core/FIRAppCheck.m:179]: I totally agree 👍 . I'll remove them.

File: FirebaseAppCheck/Sources/Core/FIRAppCheck.m [hunk 2]
```diff
@@ -172,12 +172,13 @@ - (instancetype)initWithAppName:(NSString *)appName
...
+               @" Delegate's `application(_:didFinishLaunchingWithOptions:)`"
+               @"(`application:didFinishLaunchingWithOptions:` in Objective-C)."];
```
[the remark this answers is not in the document]
Comment (ncooke3) [reply FirebaseAppCheck/Sources/Core/FIRAppCheck.m:181]: How about:
> The default FirebaseApp instance must be configured before the default FirebaseApp
instance can be initialized. One way to ensure that is to call `FirebaseApp.configure()` in the App Delegate's `application(_:didFinishLaunchingWithOptions:)` (or the `@main` struct's initializer in SwiftUI).

[the remark this answers is not in the document]
Comment (ncooke3) [reply FirebaseAppCheck/Sources/Core/FIRAppCheck.m]: I've updated the messages with a SwiftUI hint in the latest commit

**Pull request merged**